src/sentry/api/endpoints/project_details.py

```diff
@@ -274,6 +276,31 @@ def put(self, request, project):
                     'sentry:blacklisted_ips',
                     clean_newline_inputs(options['filters:blacklisted_ips'])
                 )
+            if 'filters:releases' in options:
```

Comment:
i'd also use enum in this endpoint/anywhere else you're using these strings. not sure if that's what matt meant or not, but imo, if you have one, you might as well use it literally everywhere you can. among other things, it can make it a lot easier to search for uses if someone is trying to change this behavior down the line -- `FilterTypes.RELEASES` is going to be a lot more specific than `releases`